### PR TITLE
feat: add sections for exported and imported functions

### PR DESCRIPTION
**Pull request by glimps-tdi**

### CODE REVIEW

File: elf/elf.py [hunk 1]
```diff
@@ -153,6 +153,16 @@ def add_symbols_version(self):
         # symbols_version_requirement
         pass
 
+	def add_functions(self):
+		if hasattr(self.elf, "imported_functions"):
```
Comment (gdesmar) [inline elf/elf.py:157]: Would it be possible to verify that self.elf.imported_functions and self.elf.exported_functions are not empty before creating the sections? If not, it create empty sections with empty json.
image

File: elf/elf.py [hunk 2]
```diff
@@ -153,6 +153,16 @@ def add_symbols_version(self):
         # symbols_version_requirement
         pass
 
+	def add_functions(self):
```
Comment (gdesmar) [inline elf/elf.py:156]: I believe this section of code uses tabs instead of spaces. Would it be possible to use four spaces instead per tab instead?

**Review by gdesmar — CHANGES_REQUESTED** [review]
Hi! Thank you for adding some resultsections for the elements that you consider important to show. I'm looking forward to merge your PR.

File: elf/elf.py (hunk 1 above)
Comment (glimps-tdi) [reply elf/elf.py]: Of course no problem and it's done

File: elf/elf.py (hunk 2 above)
Comment (glimps-tdi) [reply elf/elf.py]: Of course no problem and it's done

**Review by gdesmar — APPROVED** [review]

**Comment by gdesmar** [comment]
Thank you for your contribution! I'll make a new build so it should be available to everyone very soon.

**Pull request merged**